src/rulesets/oas/__tests__/valid-example.ts

```diff
@@ -1,61 +1,101 @@
 import { DiagnosticSeverity } from '@stoplight/types';
 import { RuleType, Spectral } from '../../../spectral';
-import * as ruleset from '../index.json';
+import * as ruleset from '../../oas2/index.json';
 
 // @oclif/test packages requires @types/mocha, therefore we have 2 packages coming up with similar typings
 // TS is confused and prefers the mocha ones, so we need to instrument it to pick up the Jest ones
 declare var test: jest.It;
 
 describe('valid-example', () => {
   const s = new Spectral();
-  s.setRules({
```

Comment:
setRules is a handy new way of doing things that simplifies the API a little, and addRules is deprecated now.

Also the convention here has been for `__tests__/{foo}.ts` to match the rule name it is testing, but this test file is now testing two different rules.

@P0lip can explain a bit more about the recent changes in the spectral develop branch if you are not aware of what has happened in PRs like #430.

Reply:
thanks, I don't know how to load a single rule with `loadRuleset`, is it possible @P0lip ?

Reply:
I said setRules not loadRuleset. 😅

Reply:
ah, yes, sorry. My bad